### PR TITLE
[Blog] pandas Resample

### PR DESCRIPTION
**Pull request by MarcoGorelli**
## Text styling

- [ ] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [ ] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [ ] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [ ] Blog post featured image is in PNG or JPEG format, **not** SVG.
- [ ] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/pandas-resample.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+title: "Why you probably shouldn't be using `df.resample('M')`"
+authors: [marco-gorelli]
+published: June 17, 2024
+description: 'Learn what you should be using instead'
+category: [PyData ecosystem]
+featuredImage:
+  src: /posts/pandas-resample/pandas-resample.png
+  alt: 'Image of Drake not looking impressed at `resample("M")`, and preferring `resample("MS")`'
```
Comment (MarcoGorelli) [inline apps/labs/posts/pandas-resample.md:9]: image

File: apps/labs/posts/pandas-resample.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+df.resample('MS', on='date')['price'].mean()
+```
+
+Let's learn why!
+
+## What happens when we resample by 'M' (or 'ME')
+
+Before we begin, let's talk about `'M'`. You'd be forgiven for thinking that it stands for "month".
+But actually, looking at the [pandas docs](https://pandas.pydata.org/pandas-docs/version/2.1/user_guide/timeseries.html#offset-aliases), we see that it stands for "month end"! Indeed in pandas 2.1 it was deprecated and
+renamed to `'ME'` to make its meaning clearer.
```
Comment (dharhas) [inline apps/labs/posts/pandas-resample.md:44]: You mention that `ME` means `Month End` here but I don't think you define what `MS` stands for anywhere.

File: apps/labs/posts/pandas-resample.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+- Label `'2020-01-31'` corresponds to the bucket `('2019-12-31 23:59:59.999999999', '2020-01-31 23:59:59.999999999']`
+- Label `'2020-02-29'` corresponds to the bucket `('2020-01-31 23:59:59.999999999', '2020-02-29 23:59:59.999999999']`
+- ...
+
+This looks a bit complicated, and probably not at all what users expect. Ask a crowd of data scientists which bucket
+the data point corresponding to `'2020-01-31 01:00:00'` would land in and, unless they've read this article, you'll
+probably get mixed answers.
+
+## What happens when we resample by 'MS'
```
Comment (dharhas) [inline apps/labs/posts/pandas-resample.md:79]: This might be a good place to say `MS` is `Month Start`

Comment (MarcoGorelli) [reply apps/labs/posts/pandas-resample.md:79]: done, thanks

**Review by pavithraes — APPROVED** [review]
Reads great, thank you! 🌻

**Pull request merged**